libs/vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -142,24 +154,30 @@ def _convert_to_parts(message: BaseMessage) -> List[Part]:
         return [_convert_to_prompt(part) for part in raw_content]
 
     vertex_messages = []
-    raw_system_message = None
+    convert_system_message_to_human_content = None
+    system_instruction = None
     for i, message in enumerate(history):
         if (
             i == 0
             and isinstance(message, SystemMessage)
             and not convert_system_message_to_human
         ):
-            raise ValueError(
-                """SystemMessages are not yet supported!
-                
-To automatically convert the leading SystemMessage to a HumanMessage,
-set  `convert_system_message_to_human` to True. Example:
-
-llm = ChatVertexAI(model_name="gemini-pro", convert_system_message_to_human=True)
-"""
-            )
-        elif i == 0 and isinstance(message, SystemMessage):
-            raw_system_message = message
+            if system_instruction is not None:
```

Comment:
this can't be the case since i=0, it's the first message in the list

Reply:
I see, you are capturing it below